Formula/saxon.rb

```diff
@@ -1,9 +1,9 @@
 class Saxon < Formula
   desc "XSLT and XQuery processor"
-  homepage "https://saxon.sourceforge.io"
-  url "https://downloads.sourceforge.net/project/saxon/Saxon-HE/11/Java/SaxonHE11-4J.zip"
-  version "11.4"
-  sha256 "2ec48dde4092862b1d3510d7a673d3149ad48885f8831c7878c9a85d79417094"
+  homepage "https://github.com/Saxonica/Saxon-HE"
+  url "https://github.com/Saxonica/Saxon-HE/blob/main/12/Java/SaxonHE12-0J.zip"
```

Comment:
Otherwise you're downloading the GitHub website
```suggestion
  url "https://github.com/Saxonica/Saxon-HE/blob/main/12/Java/SaxonHE12-0J.zip?raw=true"
```

Reply:
@SMillerDev Thank you!!